### DISCUSSION TITLE
SimpleLogin cancelled my subscription

### DISCUSSION THREAD

**Discussion by 4erdenko** [simple-login/app#1713] [Q&A] [answered]
![изображение](https://user-images.githubusercontent.com/93295097/235721012-aa364495-b7f5-4626-9dc9-a68c4cdd518b.png)
I didn't cancel anything myself and then all of a sudden I got a letter like this. Any idea why this happened?
Alerady sent question to support, but..

**Comment by nguyenkims** [comment]
@4erdenko our customer support has had a lot of requests recently, sorry for the delay! Can you send me your account email at hi[at]simplelogin.io so we can investigate?

**Reply by 4erdenko** [reply]
@nguyenkims 
Sure, `DELETED`

**Reply by nguyenkims** [reply]
From our record, the cancellation seems to come from you ...

**Reply by 4erdenko** [reply]
Wow!

I swear i didnt do that. Im really love this service and have no reason to cancel it. 
Maybe there are some problems with the payment service, okay, I'll just buy a new subscription closer to the end of mine, I was afraid that I could no longer be allowed to pay the service, like I was banned or something.

**Reply by nguyenkims** [reply, marked as answer]
From our Paddle dashboard, we don't see any payment issue with your subscription so I think you can wait until the current subscription ends and take a new one.
